src/usr/share/gpgmailer/gpgmailer-daemon.py

```diff
@@ -203,6 +201,7 @@ def parse_key_config(config):
     recipients_config_list = config['recipients_string'].split(',')
     recipients = []
 
+    # We won't use these recipients any more.
```

Comment:
Looking at this again, I agree this should be dead code because I think we are going to remove any notion of recipients and recipient keys from the gpgmailer configuration file. (At least for this iteration.) The only thing I think we'll have is a sender fingerprint.